modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -133,10 +146,31 @@ compute:
   replicas: 3
 ----
 endif::ipi[]
+ifdef::mapi[]
+. Add the following details to the `providerSpec` section of your machine set YAML file:
```

Comment:
```suggestion
. Add the following parameters to the `providerSpec` section of your machine set YAML file using the image details of your offer:
```
If the user is supposed to populate this sample code with the values from the previous step, you might want to either mention it in this step, or replace the values in the code block with `<value_that_you_should_replace>` type of wording. What do you think?